setup.cfg

```diff
@@ -1,3 +1,31 @@
+[metadata]
+classifiers =
+    Environment :: Console
+    License :: OSI Approved :: Apache License
+    Intended Audience :: Developers
+    Natural Language :: English
+    Operating System :: Android
+    Operating System :: Microsoft :: Windows :: Windows 10
+    Operating System :: POSIX:: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

Comment:
to be really picky we don't test for 3.9 in CI (yet)

Reply:
Really picky is definitely preferable and I hugely appreciate you reviewing the PR!

Would it be preferable to remove this classifier or add 3.9 to CI? 3.9 is still the development branch for cpython but tmk it's available on Travis

Reply:
remove for now. We can add it back later when we get into that :]